vnext/src/index.windows.js

```diff
@@ -491,7 +492,13 @@ module.exports = {
   },
 
   // Additional windows exports (Typescript components exported as flow any)
```

Comment:
Should we also add [`@deprecated`](https://jsdoc.app/tags-deprecated.html) into a JsDoc section for this api?
Or does Flow not support this?

Reply:
I'll look into that :)

Reply:
I think you'd want to change the accompanying "index.d.ts" as well, which has the Typescript definitions. I'm not sure anything will pick up the Flow ones.